src/styles/main.scss

```diff
@@ -1,3 +1,10 @@
-body {
-  margin: 0;
-}
+@import "./utils/normalize.scss";
```

Comment:
you have two different files with the same name, this one should be renamed

Reply:
Agreed, applied all the recommended changes accordingly besides the width value, as the task didn't specify the width to be set as a variable.